nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/ScanHBase.java

```diff
@@ -209,6 +209,17 @@
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
 
+    static final PropertyDescriptor BLOCK_CACHE = new PropertyDescriptor.Builder()
+            .displayName("Block Cache")
+            .name("block-cache")
+            .description("The Block Cache to enable/disable block cache on HBase scan.")
+            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
+            .allowableValues("true", "false")
+            .required(false)
+            .defaultValue("false")
```

Comment:
Since the HBase Scan object has `boolean cacheBlocks = true;` by default, do you think we should make this default to true?

If we default to false then anyone using this processor will stop using block cache without realizing it.

Also, since we are providing a default value then we can make this required. If its not required then the UI shows an option of No Value in the values list, but if you choose that and click apply, it just puts back the default. If you make it required then there is no option of No Value, just true or false.

Reply:
@bbende made changes according to your suggestions.